transform-plugins/src/main/java/io/cdap/plugin/UnionSplitter.java

```diff
@@ -31,11 +31,10 @@
 import io.cdap.cdap.etl.api.SplitterTransform;
 
 import java.nio.ByteBuffer;
-import java.util.ArrayList;
-import java.util.Collection;
-import java.util.HashMap;
-import java.util.List;
-import java.util.Map;
+import java.sql.Time;
+import java.sql.Timestamp;
+import java.text.DecimalFormat;
+import java.util.*;
```

Comment:
Don't use wildcard import. Import the needed classes one by one. You can adjust your IDE settings to do it automatically.